aips/aip-50.md

```diff
@@ -0,0 +1,65 @@
+
+### Goals
+This feature will increase the flexibility of pool owners over network/commission rates. Pool owners can adjust operator commission rates as appropriate, based on market conditions and commercial discussions. As such, this will facilitate staking pools to participate in market dynamics. I.e. the pool can be set at a lower commission to attract more stakers, or raise commission when needed. Operators can call the function themselves if they are also the pool owner, or if not, work with the pool owner they are operating for.
+
+### Out of Scope
+Operators will not be able to change the commission rate. In our existing design setup, operators are designated by pool owners.
+
+### Impact
+* *Delegation pool owners* have permission to change the operator commission rate.
+* *Operators* are impacted by commission rate changes, which will impact their rewards.
+* *Delegators* need to watch over commission rate changes so that they are not overcharged. They may assume that the commission rate is fixed.
+
+## Alternative solutions
+There is no alternative solution because the delegation pool module currently does not allow changing the operator commission rate.
+
+Moreover, we decided against allowing operators to change the commission rate, as it would not make sense in the context of our existing design setup. Right now, Operators are designated by pool owners, so eventually, pool owners control everything.
+
+## Specification
+* The maximum commission rate is uncapped (i.e., 100%). The minimum commission rate is 0%.
+* The maximum increase rate per lockup cycle is 10%. This is to safeguard delegators from sudden changes in commission rate. For example, if the current commission rate is 10%, the new commission rate cannot be higher than 20% in the next cycle.
+* There is no maximum decrease rate.
+* The commission rate change can be requested only when at least 7 days are left in the current lockup period.
+* The new commission rate will not take effect until the next lockup cycle. Multiple calls to change commission rate can be made, but only the last call before the end of the lockup cycle will take effect.
+* An event is emitted when the commission rate is changed.
+
+## Reference Implementation
+Please refer to https://github.com/aptos-labs/aptos-core/pull/10226.
+
+## Testing
+- What is the testing plan?
+Automated unit tests and testing in devnet/testnet before releasing to mainnet.
+
+## Risks and Drawbacks
+
+As a delegation pool accepts multiple delegators and delegators are provided the same stake-management API as the stake pool owner where `operator_commission_percentage` is taken from rewards generated, any changes to the operator commission rate will impact delegator staking rewards.
+1. We can mitigate risk to delegators by enforcing the maximum commission increase limit.
```

Comment:
In addition to this, this AIP also proposes a min remaining time in the lockup cycle for the commission rate to be updateable

Reply:
Added this. Thanks!